pymeasure/instruments/kepco/kepcoklp75.py

```diff
@@ -0,0 +1,163 @@
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+from enum import IntFlag
+from pymeasure.instruments import Instrument, SCPIMixin
+from pymeasure.instruments.validators import truncated_range, strict_discrete_set
+
+
+class KepcoErrorCode(IntFlag):
+    """
+    Enum for error codes during confidence testing.
+    """
+    OK = 0
+    QUARTER_SCALE_VOLTAGE = 256
+    MIN_VOLTAGE_OUTPUT = 128
+    MAX_VOLTAGE_OUTPUT = 64
+    LOOP_BACK_TEST = 32
+    DIGITAL_POT = 16
+    OPTICAL_BUFFER = 8
+    FLASH = 4
+    RAM = 2
+    ROM = 1
+
+
+class KepcoKLP75(SCPIMixin, Instrument):
+    """
+    Represents the Kepco KLP75 programmable power supply.
+
+    This class provides methods for controlling and querying the device,
+    including voltage/current setting, enabling/disabling output, applying
+    protection settings, and performing confidence tests.
+    """
+    _Vmax = 75  # Maximum Voltage (V)
+    _Imax = 16  # Maximum Current (A)
+    _Pmax = 1200  # Maximum Power (W)
+    _Imin = 0.4  # Minimum programmable current (A)
+
+    def __init__(self, adapter, name="Kepco KLP75", **kwargs):
+        """
+        Initialize the Kepco KLP75 instrument with the specified adapter and settings.
+
+        Parameters:
+            adapter: Communication adapter (e.g., GPIB address).
+            name (str): Name of the instrument.
+            kwargs: Additional keyword arguments for Instrument initialization.
+        """
+        super().__init__(
+            adapter=adapter,
+            name=name,
+            read_termination="\n",
+            write_termination="\n",
+            **kwargs
+        )
+
+    output_enabled = Instrument.control(
+        "OUTPut:STATe?", "OUTPut:STATe %d",
+        "Control whether the output is enabled or disabled.",
+        validator=strict_discrete_set,
+        values={True: 1, False: 0},
+        map_values=True,
+    )
+
+    voltage = Instrument.measurement(
+        "MEASure:VOLTage?",
+        "Measure the voltage across the output terminals (V).",
+        cast=float,
+    )
+
+    current = Instrument.measurement(
+        "MEASure:CURRent?",
+        "Measure the current through the output terminals (A).",
+        cast=float,
+    )
+
+    voltage_setpoint = Instrument.control(
+        "SOURce:VOLTage?", "SOURce:VOLTage %.1f",
+        "Set or get the voltage setpoint (0-75 V).",
```

Comment:
We use "Control" for "Set or get":
```suggestion
        "Control the voltage setpoint (0-75 V).",
```